autoscheduler/scraper/models/section.py

```diff
@@ -1,9 +1,43 @@
-from django.db import models
+"""Models for meetings and sections.
+Together, they represent all the information for a particular meeting time.
+"""
 
-class Meeting(models.Model):
-    """ Describe model here """
+from django.contrib.postgres.fields import ArrayField
+from django.db import models
+from scraper.models import Instructor
 
 class Section(models.Model):
-    """ Section will contain Meeting objects,
-        which is why its placed below it
+    """ Section contains data for a group of meetings.
+    """
+    id = models.IntegerField(primary_key=True) # id is primary key in scraped data

```

Comment:
```suggestion
    id = models.BigIntegerField(primary_key=True) # id is primary key in scraped data
```


Should be a BigIntegerField, as the ID